x-pack/plugins/security_solution/public/cases/components/add_comment/index.tsx

```diff
@@ -43,7 +43,7 @@ interface AddCommentProps {
 export const AddComment = React.memo(
   forwardRef<AddCommentRefObject, AddCommentProps>(
     ({ caseId, disabled, showLoading = true, onCommentPosted, onCommentSaving }, ref) => {
-      const { isLoading, postComment } = usePostComment(caseId);
+      const { isLoading, postComment } = usePostComment();
```

Comment:
it works but I do not understand why changing it!

Reply:
The reason I change it is because here https://github.com/elastic/kibana/pull/82996/files?file-filters%5B%5D=.ts&file-filters%5B%5D=.tsx&hide-deleted-files=true#diff-a42e73bb6513fe4fb034631921bdb628c760b2162c6a600d36964d738238dab1R44 we do not know the case id yet. The user opens all cases modal, then it selects the case she wants to attach the alert. Only then, the case id is available. At the moment the user clicked the case. I didn't want to use `usePostComment` inside the callback so I slightly change the API.

Reply:
I changed it to pass the title also along with the id.

Reply:
I see, it make sense now to me too!